docs/getting-started.md

```diff
@@ -338,7 +325,7 @@ Therefore, **during decryption** both the **sender identity and the validity of
 const decrypted = Kilt.Message.decrypt(encrypted, attester)
 ```
 
-When decryption completes, you can assume that the sender of message is also the owner.
+If the decryption process completes successfully, you can assume that the sender of message is also the owner of the claim.
```

Comment:
What is going on here? Are we verifying a signature? Only because we can decrypt the message doesn't mean that It's from the claimer?

```suggestion
If the decryption process completes successfully, you can assume that the sender of the message is also the owner of the claim.
```